### ISSUE TITLE
version 0.16.11 -> No module named 'orthopy'

### ISSUE THREAD

**Issue by RauchLukas** [sigma-py/quadpy#446]
It looks like in the latest version, there is something wrong with the `import orthopy` module. 

To recreate the Error, simply paste the two lines into google colab: 

```
!pip install quadpy
import quadpy
```

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-2-7e05a480989b>](https://localhost:8080/#) in <module>()
----> 1 import quadpy

11 frames
/usr/local/lib/python3.7/dist-packages/quadpy/c1/_gauss_kronrod.py in <module>()

ModuleNotFoundError: No module named 'orthopy'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```

Using:
```
!pip install quadpy==0.16.10
import quadpy
```

works just fine.

**Comment by dan-developer** [comment]
> It looks like in the latest version, there is something wrong with the `import orthopy` module.
> 
> To recreate the Error, simply paste the two lines into google colab:
> 
> ```
> !pip install quadpy
> import quadpy
> ```
> 
> ```
> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> [<ipython-input-2-7e05a480989b>](https://localhost:8080/#) in <module>()
> ----> 1 import quadpy
> 
> 11 frames
> /usr/local/lib/python3.7/dist-packages/quadpy/c1/_gauss_kronrod.py in <module>()
> 
> ModuleNotFoundError: No module named 'orthopy'
> 
> ---------------------------------------------------------------------------
> NOTE: If your import is failing due to a missing package, you can
> manually install dependencies using either !pip or !apt.
> 
> To view examples of installing some common dependencies, click the
> "Open Examples" button below.
> ---------------------------------------------------------------------------
> ```
> 
> Using:
> 
> ```
> !pip install quadpy==0.16.10
> import quadpy
> ```
> 
> works just fine.

Uses my repositories for temporary workaround:

https://github.com/dan-developer/dep-orthopy.git
https://github.com/dan-developer/deps-quadpy.git

**Comment by nschloe** [comment]
Alright, fixed in 0.16.13.
